robot/impl/robot_reconfigure_test.go

```diff
@@ -3101,7 +3101,7 @@ func TestInferRemoteRobotDependencyAmbiguous(t *testing.T) {
 	}
 	r.Reconfigure(ctx, reConfig)
 	utils.SelectContextOrWait(ctx, 15*time.Second)
```

Comment:
can probably lower some of these `SelectContextOrWait` then

Reply:
ah great catch, we'll see what effect that has on test duration. I couldn't lower all of them without failures, but lowered where possible.